provision/provision.go

```diff
@@ -447,6 +447,9 @@ type NodeProvisioner interface {
 	// ListNodes returns a list of all nodes registered in the provisioner.
 	ListNodes(addressFilter []string) ([]Node, error)
 
+	//ListNodesByFilters returns a list of filtered nodes by filter.
```

Comment:
Just for godoc works properly...

```suggestion
	// ListNodesByFilters returns a list of filtered nodes by filter.
```